pyHS100/cli.py

```diff
@@ -160,6 +160,7 @@ def temperature(dev, temperature):
     """Get or set color temperature. (Bulb only)"""
     if temperature is None:
         click.echo("Color temperature: %s" % dev.color_temp)
+        click.echo("(min: %s, max: %s)" % dev.valid_temperature_range)
```

Comment:
Did you test that this works (considering it's a tuple what gets returned)?

Reply:
Please also handle the case where `is_variable_color_temp` is False.

Reply:
Yes and done !